src/WebAPI/DependencyInjection.cs

```diff
@@ -32,29 +34,21 @@ public static class DependencyInjection
                         .AddFilter<ApplicationInsightsLoggerProvider>("", LogLevel.Information)
             );
         }
-
-        // TODO: Set CORS in Bicep
-        var productionCorsUrls = new string[]
-        {
-            "https://ashy-meadow-0a2bad900.3.azurestaticapps.net",
-            "https://white-desert-00e3fb600.3.azurestaticapps.net",
-            "https://rulesgpt.ssw.com.au",
-            "https://ssw.com.au/rulesgpt"
-        };
-
-        var developmentCorsUrls = new string[] { "https://localhost:5002" };
-
+        
+        var allowedCors = configuration.GetValue<string>("AllowedCORSOrigins");
+        if (allowedCors == null)
+            throw new ArgumentException("No CORS origins specified in configuration.");
+        
+        //Workaround for not being able to set arrays as variables in Azure
+        var allowedCorsList = allowedCors.Split(",");
```

Comment:
Is this clear to people modifying the string in Azure?

Reply:
Probably not but I'm not really sure how we can resolve that.
As part of #32 the value will be coming from GitHub anyway, perhaps we can put a note in the repo README?